app/services/user_services.py

```diff
@@ -0,0 +1,29 @@
+from app.exceptions.generic_exc import InvalidKeysError, InvalidTypeError
+from app.models.user_model import UserModel
+
+
+def validate_and_setattr(data: dict, user: UserModel):
+    authorized_keys = ["email", "phone", "name", "password"]
```

Comment:
I think it's better to not write this list two times, create it once in the user controller and pass as a parameter to both functions.